### PR TITLE
MGMT-12381: Remove VIP DHCP allocation defaults from test-infra

### PR DESCRIPTION
**Pull request by eliorerz**
Back on https://issues.redhat.com/browse/MGMT-10840 we changed the default value of vip_dhcp_allocation but left assisted-test-infra untouched causing vip_dhcp_allocation to stay `True` on some tests.

We need to remove vip_dhcp_allocation default from test-infra and allowing the service to set the default by itself.

/cc @osherdp

### CODE REVIEW

**Comment by eliorerz** [comment]
FYI @lalon4

File: src/assisted_test_infra/test_infra/helper_classes/cluster.py [hunk 1]
```diff
@@ -452,16 +455,18 @@ def set_advanced_networking(
         else:
             machine_networks = [models.MachineNetwork(cidr=cidr) for cidr in machine_networks]
 
+        vip_dhcp_allocation_var = {}
```
Comment (osherdp) [inline src/assisted_test_infra/test_infra/helper_classes/cluster.py:458]:
```suggestion
        extra_vars = {}
```
let's make this generic from start, I have a feeling we'll have a reason to append more to it in the future

**Review by osherdp — CHANGES_REQUESTED** [review]

File: src/assisted_test_infra/test_infra/helper_classes/cluster.py (hunk 1 above)
Comment (eliorerz) [reply src/assisted_test_infra/test_infra/helper_classes/cluster.py:458]: More generic than `extra_vars`?

**Review by osherdp — APPROVED** [review]

**Review by osherdp — APPROVED** [review]

**Pull request merged**